.github/workflows/publish.yml

```diff
@@ -63,6 +63,9 @@ jobs:
 
   check-release:
     needs: deploy
```

Comment:
I wonder if we can trigger this without triggering `depoly:` to check if it's working